src/solvers/eigsolve.jl

```diff
@@ -0,0 +1,38 @@
+
+function eigsolve_updater(
+  init;
+  psi_ref!,
+  PH_ref!,
+  outputlevel,
+  which_sweep,
+  region_updates,
+  which_region_update,
+  region_kwargs,
```

Comment:
I think `region_update_kwargs` sounds better to me.

Reply:
Though the same discussion on `exponentiate_updater` applies here, I think we should discuss how these are being passed and maybe merge them with `updater_kwargs`.

Reply:
Since this will be obsolete soon, I am in favor of sticking with `region_kwargs`.